public_html/css/index.css

```diff
@@ -153,11 +154,11 @@
 
 #top-banner {
     text-align: center;
+    margin-bottom: 20px;
```

Comment:
this keeps the header from overlapping into the subsequent section (banners or filters)